### DISCUSSION TITLE
Updated Scope for React Native Reusables

### DISCUSSION THREAD

**Discussion by mrzachnugent** [founded-labs/react-native-reusables#229] [Announcements]
Hello everyone,

We wanted to share an important update regarding the scope of the project. Originally, our goal was to create a complete shadcn implementation for React Native. However, after careful consideration, we've decided to streamline the focus of the project.

**New Scope**
The project will now include only components that are either built without third-party libraries or that use `@rn-primitives` for components originally using `radix-ui/primitives` on the web. Components that depend on other third-party libraries will not be included in this library.

**Why the Change?**
By narrowing our scope, we can ensure a more stable and consistent experience, reducing the complexity of the implementation while still providing the most essential and commonly used components.

**What’s Next?**
We believe this focused approach will allow us to deliver a high-quality toolkit that is easier to maintain and extend. We’re also planning to maintain a list of third-party libraries and repositories that can help you create any additional shadcn components that may be missing from our library. This will provide you with resources and options to supplement our toolkit as needed. We’re excited to see how the community will use and contribute to this streamlined version.

Thank you for your continued support and contributions!

Best regards,
Zach Nugent
React Native Reusables Team

**Comment by Ashraf-Ali-aa** [comment]
have a look at https://youtu.be/MKoZTAcVCJU?feature=shared expo has an experimental feature to support react web code, it could be a way to quickly migrate shadcn/ui into native. Then eventually add native implementation in subsequent iterations reducing the burden of having to constantly keep up with shadcn/ui

**Comment by GMoon777** [comment]
Love what you guys are doing, thank you for all the hard work.

**Comment by BoavistaLudwig** [comment]
Awesome, but I need the shadcn/ui Date Picker as a universal component ;-). Are you aware of anyone who is working on it?

**Comment by hungwahenry** [comment]
hey i want to contribute a haptics so buttons, toggles, selects have haptic feedback, does that fall outside this scope or nah?
